Yona/Yona/Challenges/ChallengesFirstStepViewController.swift

```diff
@@ -0,0 +1,113 @@
+
+//
+//  ChallengesFirstStepViewController.swift
+//  Yona
+//
+//  Created by Chandan on 11/04/16.
+//  Copyright © 2016 Yona. All rights reserved.
+//
+
+import UIKit
+
+class ChallengesFirstStepViewController: UIViewController,UIScrollViewDelegate {
+    
+    @IBOutlet var tegoedView: UIView!
+    @IBOutlet var tegoedBadgeLabel: UILabel!
+    
+    @IBOutlet var tijdzoneView: UIView!
+    @IBOutlet var tijdzoneBadgeLabel: UILabel!
+    
+    @IBOutlet var nogoView: UIView!
+    @IBOutlet var nogoBadgeLabel: UILabel!
+    
+    @IBOutlet var scrollView: UIScrollView!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupUI()
+        //        tegoedView.addBottomBorderWithColor(UIColor.yiWhiteColor(), width: 4.0)
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        //        //keyboard functions
+        //        let notificationCenter = NSNotificationCenter.defaultCenter()
+        //        notificationCenter.addObserver(self, selector: Selector.keyboardWasShown, name: UIKeyboardDidShowNotification, object: nil)
+        //        notificationCenter.addObserver(self, selector: Selector.keyboardWillBeHidden, name: UIKeyboardWillHideNotification, object: nil)
+        
+    }
+    
+    override func viewWillDisappear(animated: Bool) {
+        super.viewDidDisappear(animated)
+        
+        //        NSNotificationCenter.defaultCenter().removeObserver(self)
+    }
+    
+    
+    private func setSelectedCategory(categoryView: UIView) {
+        categoryView.addBottomBorderWithColor(UIColor.yiWhiteColor(), width: 4.0)
+        categoryView.alpha = 1.0
+        
+    }
+    
+    private func setDeselectOtherCategory() {
+        tegoedView.addBottomBorderWithColor(UIColor.yiPeaColor(), width: 4.0)
+        tijdzoneView.addBottomBorderWithColor(UIColor.yiPeaColor(), width: 4.0)
+        nogoView.addBottomBorderWithColor(UIColor.yiPeaColor(), width: 4.0)
+        
+        tegoedView.alpha = 0.5
+        tijdzoneView.alpha = 0.5
+        nogoView.alpha = 0.5
+    }
+    
+    private func setupUI() {
+        
+        //Nav bar Back button.
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+        UIApplication.sharedApplication().setStatusBarStyle(UIStatusBarStyle.LightContent, animated: false)
```

Comment:
@benSmith1981 it's good to set statusbar style as generic for whole app rather than in class